source/opcode.cpp

```diff
@@ -236,10 +236,10 @@ int32_t spvOpcodeIsPointer(const SpvOp opcode) {
   switch (opcode) {
```

Comment:
Moreover, since this is only used in validation, the method could move into `ValidationState_t`.